Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -97,27 +96,18 @@ public CaseDTO updateCase(long suitId, CaseDTO cs) {
 
     public void removeCases(long suitId, List<Long> caseIds) {
         Suit suit = suitDAO.getOne(suitId);
-        suit.getCases().removeIf((c) -> {
-            for (Long caseId : caseIds) {
-                if (c.getId().equals(caseId)){
-                    return true;
-                }
-            }
-            return false;
-        });
+        suit.getCases()
```

Comment:
Can we move it inside Suit?

Reply:
yep, i will do it in our new "Refactoring"-branch